src/v3/src/components/Widget/index.tsx

```diff
@@ -286,7 +286,8 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
     >
       {/* Note that we need two theme providers until we fully migrate to odyssey-mui */}
       <MuiThemeProvider theme={mapMuiThemeFromBrand(brandColors)}>
-        <ScopedCssBaseline>
+        {/* the style is to allow the widget to inherit the parent's bg color */}
+        <ScopedCssBaseline sx={{ backgroundColor: 'inherit' }}>
```

Comment:
just curious, this can't be done in the css of `AuthContainer`?

Reply:
When I tried it on the auth container, the background color of the body was still not being inherited to the widget (I initially tried to set it there).

Reply:
Curious if this behavior is what we want -- Is there a discussion about this somewhere? Or an example screenshot of how this behaves?

Reply:
So it was something I noticed when discussing the height issue with Lee. If you look at the current widget, they set the bg color in Monolith in the containing div 

<img width="1449" alt="image" src="https://user-images.githubusercontent.com/97472729/191583496-0687c008-a873-4abe-af4b-336dedeeddef.png">


If we mimic this with V3, you will notice the background color does not get applied because the MuiScopedCssBaseline div set the bg color to white. 
<img width="1534" alt="image" src="https://user-images.githubusercontent.com/97472729/191583997-d27af984-cf5e-45e5-94c6-06e18b5c85e2.png">

<img width="780" alt="image" src="https://user-images.githubusercontent.com/97472729/191584201-6977ced8-cff1-4018-8f90-e3258dbf2aab.png">

Not sure if the beta users will customize a background color or not but ultimately this is something I think we will need to support because if the widget is embedded in a company's website/page (custom hosted scenarios), the widget will set the containing bg color to white when it should inherit whatever div it is placed in). IIRC, we should not control colors/styling outside of the container, whereas this MuiScopedCssBaseline div is by setting the bg to white.

Reply:
I believe this is in parity with V2 @glenfannin-okta is that correct?

Reply:
Oh I see, thank you for the explanation - I was thinking this was affecting the inner container of the widget. Agreed that the larger surrounding container should not set its own bgcolor and instead should inherit the surrounding container